Rakefile

```diff
@@ -74,6 +74,11 @@ namespace :lint do
       sh 'mint run SwiftFormat Sources Tests Example Package.swift --config script/lint/airbnb.swiftformat --lint'
     end
   end
+
+  desc 'Lints podspec files'
+  task :pod do
```

Comment:
nit: name this task `podspec`? with `pod` it isn't quite clear what it's linting

Reply:
sure